lib/mayaUsd/ufe/MayaUsdUIInfoHandler.cpp

```diff
@@ -140,8 +140,18 @@ class _MayaIconResolver
 
     void ResetCache()
     {
-        _iconContext = PXR_NS::ArDefaultResolverContext(
-            PXR_NS::TfStringSplit(PXR_NS::TfGetenv("XBMLANGPATH", ""), ARCH_PATH_LIST_SEP));
+        auto pathVector
+            = PXR_NS::TfStringSplit(PXR_NS::TfGetenv("XBMLANGPATH", ""), ARCH_PATH_LIST_SEP);
+#ifdef __linux__
```

Comment:
I believe we are supposed to use `LINUX`
https://github.com/Autodesk/maya-usd/blob/dev/lib/mayaUsd/CMakeLists.txt#L48